rackcity/api/serializers/asset_serializers.py

```diff
@@ -256,7 +263,13 @@ def serialize_mac_addresses(network_port_model, asset):
 
 
 def serialize_network_connections(network_port_model, asset):
-    source_ports = network_port_model.objects.filter(asset=asset.id)
+   
+    try:
+        source_ports = network_port_model.objects.filter(
+            asset=asset.id, change_plan=asset.change_plan.id
+            )
+    except AttributeError:
+        source_ports = network_port_model.objects.filter(asset=asset.id)
```

Comment:
fixed serialization bug -> we need to filter by change_plan if we are filtering NetworkPortCP